src/main/java/org/apache/sysml/runtime/controlprogram/ProgramBlock.java

```diff
@@ -55,11 +57,14 @@
 	public static final String PRED_VAR = "__pred";
 	
 	protected static final Log LOG = LogFactory.getLog(ProgramBlock.class.getName());
-	private static final boolean CHECK_MATRIX_SPARSITY = false;
+	private static final boolean CHECK_MATRIX_SPARSITY = true;
```

Comment:
Please leave this disabled - it's a debugging tool and should not be active by default in master.

Reply:
Hi @mboehm7 , I am being `true`, for the Jenkins test. Can this be changed at merge time. Thanks.